examples/hetzner-deploy/oidc_login/start_client.sh

```diff
@@ -28,7 +28,7 @@ shift; shift
 shift; shift
 
 test -z "$client_name" && client_name=owncloud
-test -z "$server_url"  && server_url=https://demo.owncloud.com
+test -z "$server_url"  && server_url=https://demo.example.com
```

Comment:
a non-working default, is acceptable here for now. The script is not used currently.

Reply:
Same as above... private data in private repo please.... can be based on this repo.